### PR TITLE
Add selector in deployment config file

### PR DESCRIPTION
**Pull request by xfate123**
New to git,wish this time won't have a merge conflict

### CODE REVIEW

**Comment by merlintang** [comment]
you need to resolve the conflict, then have a better PR title.

**Comment by terrytangyuan** [comment]
You don't have to close and open a new PR. Just rebase and update this PR.

File: README.md [hunk 1]
```diff
@@ -75,10 +86,12 @@ based on your requirement.
 Following the job configuration guild in the example, you can deploy a XGBoost Job to start training or prediction like:
 ``` 
 ## For training job 
-kubectl create -f xgboostjob_v1alpha1_iris_train.yaml 
+cat config/samples/xgboost-dist/xgboostjob_v1alpha1_iris_train.yaml
```
Comment (terrytangyuan) [inline README.md:89]: Why `cat` this file here?

File: manifests/xgboost-operator/base/cluster-role-binding.yaml [hunk 2]
```diff
@@ -1,11 +1,13 @@
-apiVersion: rbac.authorization.k8s.io/v1
+apiVersion: rbac.authorization.k8s.io/v1beta1
```
Comment (terrytangyuan) [inline manifests/xgboost-operator/base/cluster-role-binding.yaml:1]: Why downgrading this?

File: README.md (hunk 1 above)
Comment (xfate123) [reply README.md:89]: Sorry, I will change the order

File: manifests/xgboost-operator/base/cluster-role-binding.yaml (hunk 2 above)
Comment (xfate123) [reply manifests/xgboost-operator/base/cluster-role-binding.yaml:1]: Sorry, that's a mistake. But I did downgrade for deployment yaml, cause when I do the installation, there's a error about deployment version, after I downgrade it, everything work fine

Comment (terrytangyuan) [reply manifests/xgboost-operator/base/cluster-role-binding.yaml:1]: What error is it?

Comment (xfate123) [reply manifests/xgboost-operator/base/cluster-role-binding.yaml:1]: error: unable to recognize "manifests/xgboost-operator/base/deployment.yaml": no matches for kind "Deployment" in version "extensions/v1beta1"
After I downgrade to apps/v1 , error disappear.
I also checked deployment.yaml on mpi operator, the version is appa/v1 and works fine

**Pull request closed without merging**